### DISCUSSION TITLE
useForm with next.js app router

### DISCUSSION THREAD

**Discussion by creib** [mantinedev/mantine#5244] [Q&A] [answered]
Hi,
I did not understand the documentation here fully.
Is it possible to make use of Mantines useForm on a next.js page using app router and rendering on the server side?
Or is it necessary to always use clientside rendering if using forms in mantine?

Thanks for a hint

**Comment by rtivital** [comment, marked as answer]
use-form uses useState hook, it requires `'use client';` directive.
